gradle/distribution.gradle

```diff
@@ -107,6 +107,7 @@ task testJavadoc(type: Javadoc) {
     classpath += sourceSets.test.compileClasspath
     classpath += sourceSets.main.compileClasspath
     source = sourceSets.test.allJava
+    options.addStringOption('Xdoclint:none', '-quiet')
```

Comment:
With this option we will ignore JavaDoc warnings, won't we? Why is it necessary?

Reply:
The quiet flag was only added for JavaDoc in TEST classes... because it started to complain A LOT